websites/R/Reddit/presence.ts

```diff
@@ -43,7 +89,7 @@ presence.on("UpdateData", async () => {
       postTitle =
         document.querySelector(
           "div._2SdHzo12ISmrC8H86TgSCp._29WrubtjAcKqzJSPdQqQ4h"
-        ) != undefined
+        ) !== undefined
```

Comment:
Shouldn't it be null? I think

Reply:
I will test again if it works or not

Reply:
https://github.com/PreMiD/Presences/pull/4238/commits/94de5614503f2e009eb26c916a737704de06b267